### PR TITLE
Update docs/ links to point to main site

### PR DESCRIPTION
**Pull request by cblecker**
**What this PR does / why we need it**:
This updates various links to either point to kubernetes.io or to the kubernetes/community repo instead of the legacy docs/ tree in k/k
Pre-requisite for #46813

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

@kubernetes/sig-docs-maintainers @chenopis @ahmetb @thockin

### CODE REVIEW

**Comment by cblecker** [comment]
I removed any changes to `docs/` so this PR can be merged by the bot.

**Comment by thockin** [comment]
Added to milestone because good links are good things.

**Comment by cblecker** [comment]
@k8s-bot pull-kubernetes-e2e-kops-aws test this
@k8s-bot pull-kubernetes-verify test this
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by cblecker** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by cblecker** [comment]
@thockin tests here are passing now. can you re-/lgtm? thanks!

**Comment by cblecker** [comment]
Got into some rebase fun due to timestamps *shakes fist*. Thankfully, #46857 will fix that. Will hold off until #46857 is merged, and then I'll rebase on top of that (hopefully for the last time)

File: examples/storm/README.md [hunk 1]
```diff
@@ -22,15 +22,15 @@ Source is freely available at:
 This example assumes you have a Kubernetes cluster installed and
 running, and that you have installed the ```kubectl``` command line
 tool somewhere in your path. Please see the [getting
-started](../../docs/getting-started-guides/) for installation
+started]\(https://kubernetes.io/docs/getting-started-guides/) for installation
```
Comment (thockin) [inline examples/storm/README.md]: why do these all have a backslash?

Comment (cblecker) [inline examples/storm/README.md]: Bad escaping on the find/replace. Fixed it, and things are looking a lot cleaner.

**Comment by thockin** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by thockin** [comment]
Federation is failing all over the place.

**Comment by cblecker** [comment]
@thockin Yeah.. sig-federation has made it non-blocking now until they can fix it.

**Comment by cblecker** [comment]
/unassign @zmerlynn @pwittrock 
@thockin: rebased (again *sigh*)

**Comment by cblecker** [comment]
@thockin rebased and all tests pass. PTAL when you have a chance :)

**Pull request merged**